_posts/2020-12-04-release-1.12.0.md

```diff
@@ -0,0 +1,332 @@
+categories: news
+authors:
+- morsapaes:
+  name: "Marta Paes"
+  twitter: "morsapaes"
+- aljoscha:
+  name: "Aljoscha Krettek"
+  twitter: "aljoscha"
+
+excerpt: The Apache Flink community is excited to announce the release of Flink 1.12.0! Close to 300 contributors worked on over 1k tickets to bring significant improvements to usability as well as new features to Flink users across the whole API stack. We're particularly excited about adding efficient batch execution to the DataStream API, Kubernetes HA as an alternative to ZooKeeper, support for upsert mode in the Kafka SQL connector and the new Python DataStream API! Read on for all major new features and improvements, important changes to be aware of and what to expect moving forward!
+---
+
+The Apache Flink community is excited to announce the release of Flink 1.12.0! Close to 300 contributors worked on over 1k tickets to bring significant improvements to usability as well as new features that simplify (and unify) Flink handling across the API stack.
+
+**Release Highlights**
+
+* The community has added support for **efficient batch execution** in the DataStream API. This is the next major milestone towards achieving a truly unified runtime for both batch and stream processing.
+
+* **Kubernetes-based High Availability (HA)** was implemented as an alternative to ZooKeeper for highly available production setups.
+
+* The Kafka SQL connector has been extended to work in **upsert mode**, supported by the ability to handle **connector metadata** in SQL DDL. **Temporal table joins** can now also be fully expressed in SQL, no longer depending on the Table API.
+
+* Support for the **DataStream API in PyFlink** expands its usage to more complex scenarios that require fine-grained control over state and time, and it’s now possible to deploy PyFlink jobs natively on **Kubernetes**.
+
+This blog post describes all major new features and improvements, important changes to be aware of and what to expect moving forward.
+
+{% toc %}
+
+The binary distribution and source artifacts are now available on the updated [Downloads page]({{ site.baseurl }}/downloads.html) of the Flink website, and the most recent distribution of PyFlink is available on [PyPI](https://pypi.org/project/apache-flink/). Please review the [release notes]({{ site.DOCS_BASE_URL }}flink-docs-release-1.12/release-notes/flink-1.12.html) carefully, and check the complete [release changelog](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12348263&styleName=Html&projectId=12315522) and [updated documentation]({{ site.DOCS_BASE_URL }}flink-docs-release-1.12/) for more details. 
+
+We encourage you to download the release and share your feedback with the community through the [Flink mailing lists](https://flink.apache.org/community.html#mailing-lists) or [JIRA](https://issues.apache.org/jira/projects/FLINK/summary).
+
+## New Features and Improvements
+
+### Batch Execution Mode in the DataStream API
```

Comment:
What do you think about adding a link to @aljoscha's FlinkForward talk on this topic? I don't know if we have precedent for such links, but it could help interested users to get more details and better understand where we're headed with the unification.